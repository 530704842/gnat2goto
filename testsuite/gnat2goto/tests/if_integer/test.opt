ALL XFAIL equality without matching types
