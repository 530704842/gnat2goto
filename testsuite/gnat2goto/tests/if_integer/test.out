[1] file divide.adb line 4 : SUCCESS
[2] file divide.adb line 6 : SUCCESS
[3] file divide.adb line 8 : FAILURE
VERIFICATION FAILED
